CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
-# v0.2.0
+# Changelog
+All notable changes to this project will be documented in this file.
 
-- Updated aws-sdk dependency to support IAM session credentials
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
When I ran the `parse-changelog.sh` script locally, it errors, indicating that:

- `https` should be `http`
- There shouldn't be a `,` at the end

```
The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
```